.github/workflows/docker.yaml

```diff
@@ -0,0 +1,48 @@
+name: Docker
+
+"on":
+  push:
+    branches:
+      - "tickets/**"
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Define the Docker tag
+        id: vars
+        run: echo ::set-output name=tag::$(echo ${GITHUB_REF} | sed -E 's,refs/(heads|tags)/,,' | sed -E 's,/,-,g')
+
+      - name: Print the tag
+        id: print
+        run: echo ${{steps.vars.outputs.tag}}
+
+      - name: Log into Docker Hub
```

Comment:
Since you're on a modernization kick here, I'll point out that Docker itself has some GitHub Actions for things like logging in, setting up the builder, and build/pushing. That could be worth looking at if you've got some time. https://docs.docker.com/ci-cd/github-actions/